config/ground-truth-yaml-files/FHIR_mappings_pcd.yml

```diff
@@ -0,0 +1,842 @@
+    class: 
+  ReactiveAirwayDx:
+    search_term: "reactive airway disease"
+    domain: Condition
+    class: ICD10 Hierarchy
+  RoadwayDistanceExposure:
+  RoadwayDistanceExposure2:
+  S177_SMOKE_100_CIG_LIFETIME:
+  SMOKE_CAT:
+  SNP1:
+  SNP2:
+  SNP3:
+  SNP4:
+  SalmeterolRx:
+    search_term: "salmeterol"
+    domain: Drug
+    vocab: RxNorm
+    class: Ingredient
+  SertralineRx:
+    search_term: "sertraline"
+    domain: Drug
+    vocab: RxNorm
+    class: Ingredient
+  Sex:
+    system: http://hl7.org/fhir/us/core/STU5/ValueSet-birthsex.html
+    search_term: 
+    domain: Patient
+    class: 
+  Sex2:
+    system: http://hl7.org/fhir/us/core/STU5/ValueSet-birthsex.html
+    search_term: 
+    domain: Patient
+    class: 
+  TLR4_AGE:
+  TLR4_AGE2:
+  TLR4_DIST_1X_cut:
+  TLR4_DIST_1X_qcut:
+  TLR4_DIST_2X_cut:
+  TLR4_DIST_2X_qcut:
+  TLR4_DIST_3X_cut:
+  TLR4_DIST_3X_qcut:
+  TamoxifenRx:
+    search_term: "tamoxifen"
+    domain: Drug
+    vocab: RxNorm
+    class: Ingredient
+  TesticularCancerDx:
+    search_term: "testicular cancer"
+    domain: Condition
+    class: ICD10 Hierarchy
+  TesticularDysfunctionDx:
+    search_term: "testicular dysfunction"
+    domain: Condition
+    class: ICD10 Hierarchy
+  TestosteroneRx:
+    search_term: "testosterone"
+    domain: Drug
+    vocab: RxNorm
+    class: Ingredient
+  TheophyllineRx:
+    search_term: "theophylline"
+    domain: Drug
+    vocab: RxNorm
+    class: Ingredient
+  TotalEDInpatientVisits:
+  TotalEDVisits:
+  TotalInpatientVisits:
+  TrazodoneRx:
+    search_term: "trazodone"
+    domain: Drug
+    vocab: RxNorm
+    class: Ingredient
+  TriptorelinRx:
+    search_term: "triptorelin"
+    domain: Drug
+    vocab: RxNorm
+    class: Ingredient
+  UterineCancerDx:
+    search_term: "uterine cancer"
+    domain: Condition
+    class: ICD10 Hierarchy
+  VenlafaxineRx:
+    search_term: "triptorelin"
+    domain: Drug
+    vocab: RxNorm
+    class: Ingredient
+  FEV1:
+    search_term: "forced expiratory volume"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  FEV1_flag_first:
+    search_term: "forced expiratory volume"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  FEV1_flag_last:
+    search_term: "forced expiratory volume"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  FVC:
+    search_term: "forced vital capacity"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  FVC_flag_first:
+    search_term: "forced vital capacity"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  FVC_flag_last:
+    search_term: "forced vital capacity"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  FEV1FVC:
+    search_term: "FEV1/FVC"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  FEV1FVC_flag_first:
+    search_term: "FEV1/FVC"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  FEV1FVC_flag_last:
+    search_term: "FEV1/FVC"
+    domain: Measurement, Observation
+    vocab: LOINC 
+  ur:

```

Comment:
@karafecho This last line with variable ```ur``` does not look right. If the file got cut off or something?

Reply:
I intentionally left a bunch of non-FHIR variables blank. This particular variables is from the US Census Bureau's ACS dataset. It gets pulled into the pipeline downstream of generating the FHIR variables. Please let me know if you'd rather have empty fields included instead.

Reply:
@karafecho Yeah, I think it'd be better to leave those non_FHIR variables out from the yaml file since they cannot be mapped anyway. Since we have a separate yaml file just for FHIR mapping, we don't need to keep the complete set of variables but rather only variables we want to map to FHIR. This will simplify the process I believe.

Reply:
Is it possible to keep the non-FHIR variables, as long as doing so won't break anything? I used all_features pcd to generate FHIR_mappings pcd, so in my mind, it makes sense to keep all variables. Plus, we can use the non_FHIR variables as a sanity check of sorts.

Reply:
Yes, it is possible. I just need to update code to filter out those empty variables. Let me approve this PR then.